### PR TITLE
Locomotion 12 6

### PR DESCRIPTION
**Pull request by avivbarniv**
## ℹ️ Change description:
Nothing here

<sub>[templates: [bug](?expand=1&template=bug.md), [feature](?expand=1&template=feature.md)]</sub>

### CODE REVIEW

File: locomotion-sdk/src/context/user/index.tsx [hunk 1]
```diff
@@ -102,7 +102,7 @@ const UserContextProvider = ({ children }: { children: any }) => {
       }
 
       auth.updateTokens(vertResponse.refreshToken, vertResponse.accessToken);
-      const userProfile = vertResponse.userProfile || {};
+      const userProfile = vertResponse.clientProfile || {};
```
Comment (OfryL) [inline locomotion-sdk/src/context/user/index.tsx]: We will be able to run local server with this one?

File: locomotion-sdk/src/pages/ActiveRide/AvailabilityVehicle/index.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        useNativeDriver={false}
+        coordinate={locationAnimated}
+      >
+        <SvgIcon svg={carIcon} />
+      </Marker.Animated>
+    </AvailabilityVehicleContainer>
+  );
+};
+
+export default AvailabilityVehicle;
```
Comment (OfryL) [inline locomotion-sdk/src/pages/ActiveRide/AvailabilityVehicle/index.tsx]: Move to comps lib

File: locomotion-sdk/src/pages/ActiveRide/index.js [hunk 3]
```diff
@@ -85,8 +86,10 @@ const RidePage = ({ menuSide, mapSettings }) => {
...
 export default props => (
   <RidePageContextProvider {...props}>
-    <RidePage
-      {...props}
-    />
+    <AvailabilityContextProvider>
+      <RidePage
+        {...props}
+      />
+    </AvailabilityContextProvider>
```
Comment (OfryL) [inline locomotion-sdk/src/pages/ActiveRide/index.js:93]: ❤️

File: locomotion-sdk/src/pages/ActiveRide/map.js [hunk 4]
```diff
@@ -124,6 +129,8 @@ export default ({
     return null;
   };
 
+  const buildAvailabilityVehicles = () => availabilityVehicles.map(vehicle => <AvailabilityVehicle location={vehicle.location} id={vehicle.id} />);
```
Comment (OfryL) [inline locomotion-sdk/src/pages/ActiveRide/map.js:132]: i think this better be inline cause memo etc,
add key?

**Review by OfryL — APPROVED** [review]

File: locomotion-sdk/src/pages/ActiveRide/map.js (hunk 4 above)
Comment (avivbarniv) [reply locomotion-sdk/src/pages/ActiveRide/map.js:132]: I have key inside the copmonent, I'll add memo

File: locomotion-sdk/src/context/user/index.tsx (hunk 1 above)
Comment (avivbarniv) [reply locomotion-sdk/src/context/user/index.tsx]: Only when using the new backend, I'll revert for now

**Pull request merged**